MetaMask/metamask-extension
# Add Account Snap MetaMetrics data to transactions and signatures

## **Description**

Closes MetaMask/accounts-planning#71
Paired with Consensys/segment-schema#122

## **Manual testing steps**

1. Build from this branch or download the Flask build from this PR
2. Turn on MetaMetrics
3. In the Extensions page, open "Inspect views background.html"
4. Go to the "Network" tab
5. Filter by the word `batch`
6. Look for Signature and Transaction MetaMetrics events that come through when you perform actions
7. If you have access to Mixpanel, look at Mixpanel to make sure the events come in

## **Pre-merge author checklist**


## **Pre-merge reviewer checklist**
